guides/user/get-started/index.md

```diff
@@ -0,0 +1,117 @@
+---
+layout: single
+title:  "Get Started Using Spinnaker"
+sidebar:
+  nav: guides
+---
+
+{% include toc %}
+
+> This article assumes you are new to Spinnaker. It contains basic information
+> about what to do after [installing Spinnaker](/setup/install/).
+
+## Before you get going...
+
+Whether you're an operator creating or managing a Spinnaker deployment, or an
+end user getting started with Spinnaker, here are some pointers to get you
+started.
+
+### Operators
+
+* If you want to install Spinnaker, [here you go](/setup/install/)
+
+* If you want to find out more about tuning and maintaining Spinnaker, here are
+some things to check out:
+
+  - [Advanced configuration](/setup/other_config/)
+  - [Productionize Spinnaker](/setup/productionize/)
+
+### Users
+
+* If you just want to try out Spinnaker, here are some
+[quickstarts](/setup/quickstart/)
+
+* If Spinnaker is already installed in your org, and you want to practice it
+using some guided tutorials, [here are some
+codelabs](/guides/tutorials/codelabs/)
+
+* If you want a very basic overview of how to use Spinnaker, read on...
+
+## Using Spinnaker: the high-level process
+
+<!--
+Notes:
+* For step 2, this isn't quite right: many people are going to want to use
+pipelines to create infrastructure. Creating infrastructure manually, using the
+ui, is optional, and I can go straight to creating a pipeline.
+
+So, it's more like
+
+1. Create an application
+2. Do all of this other stuff
+(unordered list...)
+* pipelines
+* create infrastructure, which can be done inside the pipeline
+* deploy
+
+
+Further, examine who is reading this.
+Right now I'm, possibly erroneously, jumping here from the end of install. But
+this page is for *users*, not operators. In most cases it's the operators who
+will install, and the devs who will use. There can be devops overlap, but we
+need to keep the roles distinct.
+This can be more fully solved when I create a separate ops guide*
+Mean time, I need to somehow make this delineation clearer.
+
+
+
+* some of the things lw mentioned that operators will/might be doing, post
+install...
+  - setting up pipeline templates
+  -
+
+-->
+
+
+
+1. [Create an application](/guides/user/applications/)
+
+   Typically, you'll have one application per microservice.
```

Comment:
Usually a bunch of microservices end up _comprising_ or are _composed_ into one application.

Reply:
I think in spinnaker it's fair to map a spinnaker application to a microservice. It's not how you _need_ to do it, but it's the pattern we've seen.